catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -204,7 +204,7 @@
         <dependency>
             <groupId>org.eclipse.jetty</groupId>
             <artifactId>jetty-servlets</artifactId>
-            <version>${jetty.version}</version>
+            <version>9.2.19.v20160908</version>
```

Comment:
@lessarderic Something along the lines of "Temporarily adding deprecated GzipFilter functionality until GzipHandler supported by PaxWeb"?